### ISSUE TITLE
Use level-according stats for equipment

### ISSUE THREAD

**Issue by 1caiser** [fribbels/Fribbels-Epic-7-Optimizer#51]
I noticed that gear main stats are not calculated according to its level, and instead assumes all pieces of gear are at +15 for main stat calculations. I figured they would be statted according to level, seeing as the Optimizer imports gear with levels also logged.

**Comment by fribbels** [comment]
That is an intended feature, so that non +15 gear can be used in the optimization calculation without the penalty of the main stat difference. Otherwise they would be missing so much from the main stat that they would never get selected

**Comment by 1caiser** [comment]
Yep, I now see that there was the exact same ticket in the Closed section. Thank you for developing the app!
